captum/attr/_core/layer/layer_integrated_gradients.py

```diff
@@ -258,8 +271,12 @@ def attribute(
 
         # inputs -> these inputs are scaled
         def gradient_func(
-            forward_fn, inputs, target_ind=None, additional_forward_args=None
+            forward_fn: Callable,
+            inputs: Union[Tensor, Tuple[Tensor,...]],
+            target_ind: int = None,
+            additional_forward_args: Any = None
         ):
+            scattered_inputs: Union[Tuple[Union[Tensor, Tuple[Tensor, ...]]], Union[Tuple[Tensor, ...], List[Tuple[Any, ...]]]]
```

Comment:
I think we can avoid having this extra declaration, and instead just add "# type:ignore" to line 283 below with a short comment above on why this is necessary. This seems to be caused by the scatter method not having a precise enough return type in its stub, so let's just suppress the type warning there.

Reply:
Agreed, done.